includes/packages/Manuel-Shipment/manual-shipment.php

```diff
@@ -0,0 +1,297 @@
+								'logo'=>'http://placehold.it/120x120&text=image1',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								],
+							'TNT' => [
+								'name'=>'TNT',
+								'logo'=>'http://placehold.it/120x120&text=image2',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								]
+							];
+
+		$this->shipping_carriers = $this->get_shipping_providers();
+
+		// Action to register custom order status
+		add_action( 'init', [$this,'hez_register_shipped_order_status'] );
+
+		// Add custom status to order status list
+		add_filter( 'wc_order_statuses', [$this,'hez_add_shipped_to_order_statuses'] );
+
+		// Add Custom fields for woocommerce order
+		add_action( "add_meta_boxes", [$this,"hez_add_meta_boxes"] );
+		
+		// Ajax handler function
+		add_action( 'wp_ajax_hezarfen_shipping_data_action', [$this,'hezarfen_shipping_data_action'] );
+
+		// Add Custom fields for woocommerce order
+		add_action( 'wp_ajax_nopriv_hezarfen_shipping_data_action', [$this,'hezarfen_shipping_data_action'] );
+		
+		// Show Shipping fields in my account
+		add_action( 'woocommerce_order_details_after_order_table', [$this,'hezarfen_show_shipping_data'] );
+		
+	}
+
+	private function get_shipping_providers(){
+		return apply_filters( 'hezarfen_supported_shipping_carriers', $this->shipping_carriers );
+	}
+
+	// Function for register custom order status
+	public function hez_register_shipped_order_status() {
+	    register_post_status( 'wc-hez_shipped', array(
+	        'label'                     => 'Kargolandı',
+	        'public'                    => true,
+	        'exclude_from_search'       => false,
+	        'show_in_admin_all_list'    => true,
+	        'show_in_admin_status_list' => true,
+	        'label_count'               => _n_noop( 'Kargolandı <span class="count">(%s)</span>', 'Kargolandı <span class="count">(%s)</span>' )
+	    ) );
+	}
+
+	// Add custom status to order status list
+	public function hez_add_shipped_to_order_statuses( $order_statuses ) {
+	    $new_order_statuses = array();
+	    foreach ( $order_statuses as $key => $status ) {
+	        $new_order_statuses[ $key ] = $status;
+	        if ( 'wc-processing' === $key ) {
+	            $new_order_statuses['wc-hez_shipped'] = 'Kargolandı';
+	        }
+	    }
+	    return $new_order_statuses;
+	}
+	// Add Custom fields for woocommerce order
+	public function hez_add_meta_boxes()
+	{
+		if(isset($_GET['post'])){
+			$order_id=$_GET['post'];
+		    add_meta_box(
+		        "woocommerce-order-my-custom",
+		        __( "Shipping Options" ),
+		        [$this,"hez_order_shipping_options"],
+		        "shop_order",
+		        "side",
+		        "default"
+		    );		
+		}		
+	}
+	// Add Custom fields for woocommerce order
+	public function hez_order_shipping_options(){
+	    global $woocommerce,$wpdb;
+	    $shipping_carriers=$this->shipping_carriers;
+	    $order_id=$_GET['post'];
```

Comment:
Please sanitize the data

https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/